src/navigate-pop/tiling-stacking-windows.md

```diff
@@ -0,0 +1,86 @@
+# Manipulating Windows
+
+This section describes the Auto Tiling, Stacking, and window adjustment features in Pop!\_OS.
+
+---
+## Window Adjustment Mode
+
+Adjustment mode allows users to easily manipulate the placement of windows using keyboard shortcuts.
+
+Place a window into Management mode by positioning your cursor over the window, and then pressing `SUPER` + `Enter`. The window will be highlighted in yellow. You can now manipulate the window with `🠐` `🠑` `🠒` `🠓` keys in combination with other keys.
+
+![Adjustment Mode Manipulation](/images/tiling-stacking-windows/adjustment-mode-manipulation.gif)
+
+All keyboard shortcuts available in adjustment mode can be viewed and modified in Settings ➞ Keyboard ➞ View and Customize Shortcuts ➞ Move, resize, and swap windows.
+
+## Auto Tiling Windows
+
+Pop!\_OS optimizes workflows using a smart window management system called Auto Tiling. Auto Tiling automatically positions and sizes windows.
+
+![Auto Tiling](/images/tiling-stacking-windows/auto-tiling.png)
+
+Enable Auto Tiling by clicking on the tile icon in the upper right corner and toggling the button. Tiling settings include options for window titles, active window hint, active window color, and gaps.
+
+![Enable Autotiling](/images/tiling-stacking-windows/enable-autotiling.png)
+
+| Option                 | Function |
+|----------------------|--------|
+| Tile Windows| Toggles the Auto Tiling feature on or off  |
+| Floating Window Exception  | Allows adding exceptions to the Auto Tiling feature allowing free positioning of specified windows.|
+| Toggle Tiling          | Enable/Disable the Auto Tiling feature with a keyboard shortcut. |
+| Show Active Hint       | Highlights the selected window with a colored border. |
+| Gaps                | Sets gap spacing between tiled windows. |
+
+All keyboard shortcuts for Auto Tiling mode can be found in Settings ➞ Keyboard ➞ View and Customize Shortcuts ➞ Tiling.
+
+### Add a Floating Window Exception
+
+Floating Window Exceptions exclude individual windows from Auto Tiling. Pop!\_OS applies default exceptions to some applications based on user feedback.
+
+1. Run the application that needs to be exempted from Auto Tiling. Click the tile menu, then click `Floating Window Exceptions`.
+
+    ![Floating Window Exceptions](/images/tiling-stacking-windows/floating-window-exceptions.png)
+
+2. Click `Select` to add an exception.
+
+    ![Click Select](/images/tiling-stacking-windows/click-select.png)
+
+3. Select the running application that should be exempted from Auto Tiling.
+
+    ![Select Exempted App](/images/tiling-stacking-windows/select-exempted-app.png)
+
+4. Choose to create an exception for all windows belonging to an application, or only the currently selected window. In this example, only the current Terminal window will be selected for exemption.
```

Comment:
This might not be entirely clear to someone who doesn't know the system already. The "Current Window Only" option uses the window's title to apply the exception in addition to its application, so any windows with the same title (now or in the future) will all have the exception applied.